### ISSUE TITLE
Provide Alternative text for images

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#51]
## Description
All images must provide alternative text. If images provide additional meaningful information that the surrounding text doesn't already convey, provide an "alt" attribute with meaningful text that describes the image.

### Header/Footer

- [ ] Add support in the CMS for providing alt text to the Funding Logos global
- [ ] Render provided alt text for funding logos image

![Screen Shot 2022-02-03 at 9 53 15 AM](https://user-images.githubusercontent.com/8166060/152400295-0a3430eb-ed78-4f9b-bade-d79ef2cc9934.png)

### Home

- [x] Hompage hero image is decorative in this context, and therefore should have `alt=""` and be hidden from screen readers using `aria-hidden="true"` or `role="presentation"`

### "Featured" Images

- [ ] Pages' "featured" images are decorative and therefore should have `alt=""` and be hidden from screen readers using `aria-hidden="true"` or `role="presentation"`.

### Events

- [x] Remove unnecessary title on lock svg icon.  Hide from screen readers using `aria-hidden="true"` or `role="presentation"`.

![0fdca7993353ab7f566fd83e1035256c6c63be09-1641287766](https://user-images.githubusercontent.com/8166060/152437808-7af0fe1c-f1d2-4a29-9e19-863536aa227e.png)

- [x] Event thumbnails are decorative and therefore should have `alt=""` and be hidden from screen readers using `aria-hidden="true"` or `role="presentation"`.

![42c1fd52548a99fda9b2864b0fc901e8d89235df-1641287243](https://user-images.githubusercontent.com/8166060/152439187-55c8f64f-1d85-474b-9e7f-ebd5f6bf90fa.png)

### Contact Links' Icons

- [x] Mail and phone icons are decorative and therefore should have `alt=""` and be hidden from screen readers using `aria-hidden="true"` or `role="presentation"`.  Rubin content editors will edit text of address and phone fields to include indication of what the address and number are.

![f3b735f74de3ca0eb673d9319145236b5af9c0ae-1641276122](https://user-images.githubusercontent.com/8166060/152439252-da17ac05-045c-4b11-af33-26230455128d.png)

## CIC Estimate
1 point

**Comment by dananjohnson** [comment]
@blnkt I'm trying to determine what "Featured" images refer to. We have this field on page entries, but it's just used for OpenGraph tags in the HTML `<head>`. Is there another field I should be looking at?

<img width="392" alt="Screen Shot 2022-05-24 at 11 47 59 AM" src="https://user-images.githubusercontent.com/12454203/170110271-3fb9439b-759d-4754-bab2-ce5b210c357d.png">

**Comment by blnkt** [comment]
@dananjohnson You're right that's not super clear of me.  It's been a while now since I wrote/looked at these, but I believe this is in reference to the images used in gridBlocks like, for instance, the Related Items CTA grid.  I think it was my guess that those images in the grids were pulling from the featured images.  But perhaps I misjudged from whence they are sourced.  Either way I think my thought was that images in this tile/thumbnail context were purely decorative and as such could be excluded from screen readers.  Thoughts?

**Comment by dananjohnson** [comment]
@blnkt welcome back! OK that makes sense now, thanks. The commit referenced above (0995cf3) should take care of that.
